pkg/transport/process/extension/uds_executable.go

```diff
@@ -0,0 +1,107 @@
+package extension
+
+import (
+	"context"
+	"fmt"
+	"io"
+	"net"
+	"os"
+	"os/exec"
+	"time"
+
+	"github.com/gardener/component-cli/pkg/transport/process"
+	"github.com/gardener/component-cli/pkg/utils"
+)
+
+const serverAddressFlag = "--addr"
```

Comment:
I'm not sure if we should really use a flag. Flags are parsed differently depending on the language/framework.
I propose to start with a env var.